cedar-testing/tests/cedar-policy-cli/main.rs

```diff
@@ -82,32 +80,6 @@ fn perform_integration_test_from_json(jsonfile: impl AsRef<Path>) {
     let schema_file = resolve_integration_test_path(&test.schema);
 
     for json_request in test.requests.into_iter() {
-        let policies_text = std::fs::read_to_string(policy_file.clone())
-            .unwrap_or_else(|e| panic!("error loading policy file {}: {e}", &test.policies));
-        let policies_res = PolicySet::from_str(&policies_text);
-        // If parsing fails we don't want to quit immediately. Instead we want to
-        // check that the expected decision is "Deny" and that the parse error is
-        // of the expected type (NotAFunction).
-        if let Err(parse_errs) = policies_res {
-            // We may see a `NotAFunction` parse error for programmatically generated
-            // policies, which are not guaranteed to be parsable
-            assert_eq!(
-                json_request.decision,
-                Decision::Deny,
-                "test {} failed for request \"{}\" \n Parse errors should only occur for deny",
-                jsonfile.display(),
-                &json_request.description
-            );
-            assert!(
-                parse_errs
-                    .iter()
-                    .any(|e| e.to_string().ends_with("not a function")),
-                "unexpected parse errors in test {}: {}",
-                jsonfile.display(),
-                parse_errs,
-            );
-            continue;
-        };
```

Comment:
Nice.  Does this mean that programmatically generated policies are now guaranteed to be parseable?

Reply:
Afaik, the issue still exists in our generators. The difference is that we no longer save invalid policy files to the generated integration tests. See this function: https://github.com/cedar-policy/cedar-spec/blob/8b6aa6956ca0720de51c4dbe710d1b162648f1e0/cedar-drt/fuzz/src/dump.rs#L196